.ipynb_checkpoints/artificial_cart_builder-checkpoint.ipynb

```diff
@@ -0,0 +1,84 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import codecs\n",
+    "import json\n",
+    "from bs4 import BeautifulSoup\n",
+    "import random\n",
+    "\n",
+    "def build_cart(x):\n",
+    "    try:\n",
+    "        # html from a random article with 28 Amazon items \n",
+    "        our_page = codecs.open('amazon-items1.html', 'r') \n",
+    "        soup = BeautifulSoup(our_page, 'html.parser')\n",
+    "\n",
+    "        # all products are within this div tag\n",
+    "        products = soup.findAll(\"div\",{\"class\":\"slide\"})\n",
+    "\n",
+    "        # initialize dictionary for all product info\n",
+    "        appendix={}\n",
+    "\n",
+    "        # loop through all products in the cart and get their link, item code, name, price\n",
+    "        for each in products:\n",
+    "            link = each.a['href']\n",
+    "            asin = each.a['data-amzn-asin']\n",
+    "            split = each.h3.text.split(\", \")\n",
+    "            name = split[0]\n",
+    "            price = split[1]\n",
+    "\n",
+    "            appendix.update({\n",
+    "                asin:{\n",
+    "                    \"link\" : link,\n",
+    "                    \"name\" : name,\n",
+    "                    \"price\" : price,\n",
+    "                    # \"category\" : category\n",
+    "                }\n",
+    "            })\n",
+    "\n",
+    "        # method for building cart; parameter must be a non-negative integer less than or equal to 28  \n",
```

Comment:
i like these conditions you've added to ensure that the cart number is valid! this will be very helpful when our web-scraping is integrated with everything
also great work on the quick turnaround !!